### ISSUE TITLE
Offline support for caniuse.com

### ISSUE THREAD

**Issue by CaesarSchinas** [Fyrd/caniuse#1756]
I work offline a lot (traveling etc) and it would be really great to have caniuse.com work offline.
I presume this would be useful for many other people too.

I imagine this would not be too difficult to implement since the site is already effectively a single-page application. That said, I have not looked at how data is loaded on the page so perhaps it could be harder than I am assuming...

**Comment by tomByrer** [comment]
What do you picture @CaesarSchinas?  "Application Cache" so the browser keeps the app in memory (hopefully; remember that mobile devices & even desktop browsers have limits.).  A Chrome Store / Firefox Marketplace app?  Downloadable `caniuse.tar.gz` file?

**Comment by CaesarSchinas** [comment]
@tomByrer, yes, I had been thinking of AppCache for the site and perhaps IndexedDB for the data.
Do you think there would be a problem with reaching browser storage limits? The dataset isn't *that* big is it?

An alternative which would also be nice would be a Dash docset.

**Comment by CaesarSchinas** [comment]
Incidentally devdocs.io seems to manage to store quite a lot of data locally, we could look at that for inspiration.

**Comment by CaesarSchinas** [comment]
Or (sorry for string of comments in quick succession) maybe collaborate with devdocs.io to get a caniuse docset on there?

**Comment by tomByrer** [comment]
devdocs.io seems like a good idea.  Seems like you have to configure their scrapper, but I wonder if submitting the .json files directly with a template would be better?

(BTW, you can edit old posts; I often do.)

**Comment by lastpotato** [comment]
I've posted a [suggestion](https://trello.com/c/PVnfdeaN/26-suggest-new-docs-here#comment-56d2999f186ead1e99c37ee2) in [devdocs's trello board](https://trello.com/b/6BmTulfx/devdocs-documentation) with link to this issue. Despite the fact that they require >3k stars for project to be included, I hope it will be implemented anyway.

**Comment by lastpotato** [comment]
It was implemented (thanks to @Thibaut) and support tables is now available in [devdocs.io](https://devdocs.io) (can also be accessed offline).

**Comment by GitToTheHub** [comment]
Could caniuse maybe download the data like devdocs for offline? I saw the data is available as a "data.json" in this repo, maybe it could be used as offline data?
